lighthouse-core/computed/metrics/responsiveness.js

```diff
@@ -42,20 +67,86 @@ class Responsiveness {
     return responsivenessEvents[index];
   }
 
+  /**
+   * Finds the interaction event that was probably the responsivenessEvent.maxDuration
+   * source.
+   * Note that (presumably due to rounding to ms), the interaction duration may not
+   * be the same value as `maxDuration`, just the closest value. Function will throw
+   * if the closest match is off by more than 4ms.
```

Comment:
nit: instead of throw, maybe return null? would force caller to be sure to handle it

or is there a reason you'd prefer this computed artifact to error when no interaction is present?

Reply:
compute_ already returns a nullable value, so might as well make this function do that too?

Reply:
This shouldn't be a thing that happens, even the 4ms difference needs to be fixed upstream (something weird is happening with rounding before the trace events are emitted), so I'm ok calling it a fatal case.

FWIW I've never hit a case of 5ms, so I don't think this should happen in practice except in actually exceptional cases.